app/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+
+app.use(express.static(path.join(__dirname, '..', 'dist')));
+
+app.get('/preferences.js', function (req, res) {
```

Comment:
I'm probably to blame for this name, but come to think of it, preferences makes it sound too much like user preferences. We should probably call this `runtime-config.js` like that Medium blog post you shared.